gulpfile.js

```diff
@@ -488,8 +488,19 @@ gulp.task('generate-icon-names', done => {
       .join('')
       .replace('Ios', 'iOS');
 
-  const templateData = entries =>
-    Object.assign(...entries.map(([k]) => ({ [k]: codify(k) })));
+  // Once we drop support for the legacy API, we can leave the suffix in the string
+  // so that `BPKIcon.m` doesn't need to add it back in programmatically
```

Comment:
For now while we support both API's I think it's best to cut off the suffix so both API's can use the same underlying mechanism. We can clean up the code in a few weeks when we only need to support the new API.